src/templates/actors/ship-sheet.html

```diff
@@ -426,7 +426,13 @@
                   <input
                     class="w-24 text-center px-1.5 py-1.5 border border-gray-800 bg-gray-400 bg-opacity-75 placeholder-blue-800 placeholder-opacity-75 rounded-md"
                     type="number" min="0" name="data.amountOwed" value="{{actor.data.data.amountOwed}}" />
-                  <label>{{localize 'swnr.sheet.ship.amount-owed'}}</label>
+                  <label>{{localize 'swnr.sheet.ship.amount-owed'}}
+                    <a class="credits-change text-gray-100 hover:text-green-400 transition-colors"
+                    title="{{localize 'swnr.sheet.credit-change'}} {{actor.data.data.tweak.debtDisplay}}"
```

Comment:
The second var will need to change as the tweak attribute is just for actors.

Reply:
you mean {{actor.data.data.tweak.debtDisplay}} ?

Reply:
yeah that was the field that allowed for custom labels of the 'debt" currency